### PR TITLE
Adding CI for FE

### PR DESCRIPTION
**Pull request by ericrkuo**
https://github.com/ubclaunchpad/PromoPal-backend/issues/92

### CODE REVIEW

**Comment by ericrkuo** [comment]
@andrewzulaybar here's some really quick CI for FE. Don't forget to configure the repo settings for the option to update with latest code from main (master)

**Comment by ericrkuo** [comment]
Surprisingly, the CI for the front end takes a bit longer than backend CI 😅 I think it's because it takes a while to run the `yarn build` command for react

File: package.json [hunk 1]
```diff
@@ -59,7 +60,7 @@
   },
   "lint-staged": {
     "*.{ts,tsx}": [
-      "yarn lint",
+      "yarn lintFix",
```
Comment (andrewzulaybar) [inline package.json:63]: Ahhh yes I like this better thanks!

File: src/services/GooglePlacesService.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import axios, { AxiosError, AxiosResponse } from 'axios';
 import * as dotenv from 'dotenv';
-import { RestaurantDetails } from '.././types/RestaurantDetails';
+import { RestaurantDetails } from '../types/RestaurantDetails';
 import { RestaurantInfo } from '../types/RestaurantInfo';
 
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```
Comment (andrewzulaybar) [inline src/services/GooglePlacesService.ts:6]: Thanks for adding these, I just created an issue to remind myself to handle these later: https://github.com/ubclaunchpad/PromoPal-frontend/issues/2

**Review by andrewzulaybar — APPROVED** [review]
Looks good to me!

**Review by lilyydu — APPROVED** [review]
Looks awesome! Thanks for adding it 🥳

**Pull request merged**